config/locales/publishers.yml

```diff
@@ -183,25 +191,45 @@ en:
               content_with_no_link: You can choose for candidates to apply for the job through Teaching Vacancies.
               link: Preview the application form
           enable_job_applications_html: >-
-            Candidates can apply using the Teaching Vacancies application form,
-            or you can direct them to another service, like the local authority or a school's website.
-            Read about %{link}.
+            <p class="govuk-body">This application form:</p>
+            <ul class="govuk-list govuk-list--bullet">
+            <li>provides a quick and simple experience for applicants</li>
+            <li>makes it easy for you to view, shortlist or reject applications in one place</li>
+            <li>is always up to date with the 'keeping children safe in education' guidelines</li>
+            </ul>
```

Comment:
not really the ideal way to do this, putting this markup in yml. there is a better way, discuss tomorrow

Reply:
in fact i made change so nothing to do. you can use this syntax and iterate over it in template. usually best to avoid putting anything but text in yml files if possible to keep things clean. occasionally there is no option, but in this case there is

Reply:
```          about_items:
            - provides a quick and simple experience for applicants
            - makes it easy for you to view, shortlist or reject applications in one place
            - is always up to date with the ‘keeping children safe in education’ guidelines```